lib/models/billing.coffee

```diff
@@ -0,0 +1,222 @@
+###
+Copyright 2016 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+###
+
+{ findCallback, onlyIf } = require('../util')
+
+getBillingModel = (deps, opts) ->
+	{ request } = deps
+	{ apiUrl, isBrowser } = opts
+
+	###*
+	# @summary Get the user's billing account
+	# @name getAccount
+	# @public
+	# @function
+	# @memberof resin.models.billing
+	#
+	# @fulfil {Object} - billing account
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.billing.getAccount().then(function(billingAccount) {
+	# 	console.log(billingAccount);
+	# });
+	#
+	# @example
+	# resin.models.billing.getAccount(function(error, billingAccount) {
+	# 	if (error) throw error;
+	# 	console.log(billingAccount);
+	# });
+	###
+	exports.getAccount = (callback) ->
+		callback = findCallback(arguments)
```

Comment:
You only need this `findCallback` line (here and in all the functions below) when you have an optional argument before the callback, so it's not clear from the parameter positions which argument is the callback.

Here, with only one parameter, you can just drop it.